tests/authenticated.spec.js

```diff
@@ -609,6 +609,35 @@ suite('AuthenticatedClient', () => {
       .catch(err => assert.ifError(err) || assert.fail());
   });
 
+  test('.getFees()', done => {
+    const expectedResponse = {
+      maker_fee_rate: '0.0050',
+      taker_fee_rate: '0.0050',
+      usd_volume: '500.00',
+    };
+
+    nock(EXCHANGE_API_URL)
+      .get('/funding')
+      .times(2)
+      .reply(200, expectedResponse);
+
+    let cbtest = new Promise((resolve, reject) => {
+      authClient.getFundings((err, resp, data) => {
+        if (err) {
+          reject(err);
+        }
+        assert.deepEqual(data, expectedResponse);
+        resolve();
+      });
+    });
+
+    let promisetest = authClient.getFundings();
```

Comment:
`getFundings` should be replaced with `getFees `

Reply:
Oh gosh, sorry, I  saw your comments and couldve swore I swapped everything out with `fees`. So Iwent to my editor, and noticed I didnt save. Sorry about that! Should be all fixed now.